recipes/boostdep/all/conanfile.py

```diff
@@ -0,0 +1,63 @@
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class BoostDepConan(ConanFile):
+    name = "boostdep"
+    settings = "os", "arch", "compiler", "build_type"
+    description = "A tool to create Boost module dependency reports"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/boostorg/boostdep"
+    license = "BSL-1.0"
+    topics = ("conan", "boostdep", "dependency", "tree")
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def requirements(self):
+        self.requires("boost/1.70.0")
```

Comment:
```suggestion
        self.requires("boost/1.73.0")
```

boostdep is 1.73, but its requirement is 1.70 ?

Reply:
boostdep is not really using latest boost features.
I lowered the version to enable bootstrapping for a newer boost version,
but I think it does not matter as you can always override the version.

Reply:
Probably users will override it. Aligning the version will avoid the warning message related to version overriding.

Reply:
Or that was the reason: https://github.com/conan-io/conan-center-index/pull/2096#issuecomment-661325542
:laughing:

Reply:
:facepalm:  let's revert and keep 1.70